README.md

```diff
@@ -41,11 +41,11 @@ _Note_: If you already closed the notification, you can find a "Develop your pro
 
 If you want to view non-public repositories, you need to add an OAuth token. The token is stored only in your browser, and only send to GitHub when fetching your repository's files. Click on the icon near the bottom of the left-hand row of icons, and the dialog box will prompt you for it, and even take you to your GitHub settings page to generate one, if needed.
 
-<img height="500px" src="https://raw.githubusercontent.com/timbaileyjones/github1s/master/resources/images/auth-token.png" />
+<img height="500px" src="https://raw.githubusercontent.com/conwnet/github1s/master/resources/images/auth-token.png" />
 
 ## Screenshots
 
-![VS Code - GitHub1s](https://raw.githubusercontent.com/mohitjaisal/github1s/master/resources/images/Github1sDemo1.gif)
+![VS Code - GitHub1s](https://raw.githubusercontent.com/conwnet/github1s/master/resources/images/GitHub1sDemo1.gif)
```

Comment:
This changes the images to rely on the canonical repository location.

It's also changing the filename to match the brand (this can potentially cause problems for Windows users as case changes for files can be messy).